SubmittedAssignments/RohitBagade_Assignment74/css/style.css

```
@@ -0,0 +1,499 @@
+    border-style: solid;
+	left: -25mm;
+    border-width: 15mm 15mm 10mm 10mm;
+	border-color: #0077be transparent transparent transparent;
+	z-index: 63;
+	transform: rotateZ(-40deg);
+    top: 70mm;
+    width: 155mm;
+    height: -2mm;
+    transform-origin: left center;
+    margin: 5mm;
+    border: 7mm solid #ffffff;
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-style



(lint/suspicious/noShorthandPropertyOverrides)